phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/LeftButton.java

```diff
@@ -29,7 +29,7 @@ public LeftButton(String name) {
         super(name);
         this.name = name;
         this.getStyleClass().add("leftButton");
-        this.setPrefWidth(Double.MAX_VALUE);
+        this.setMaxWidth(Double.MAX_VALUE);
```

Comment:
Isn't this the default max width?
In any case I would prefer to see these definitions in the css files.